modules/core/src/main/scala/scala/build/internals/EnvsUtil.scala

```diff
@@ -0,0 +1,135 @@
+package scala.build.internals
+
+import scala.build.internals.ConsoleUtils.ScalaCliConsole
+
+object EnvsUtil {
+  def helpMessage(isPower: Boolean): String =
+    s"""The following is the list of environment variables used and recognized by Scala CLI.
+       |It should by no means be treated as an exhaustive list
+       |Some tools and libraries Scala CLI integrates with may have their own, which may or may not be listed here.
+       |${if isPower then "" else "For the expanded list, pass --power." + System.lineSeparator()}
+       |${
+        val maxFullNameLength =
+          EnvVar.all.filter(!_.requiresPower || isPower).map(_.name.length).max
+        EnvVar.allGroups
+          .map(_.subsectionMessage(maxFullNameLength, isPower))
+          .filter(_.linesIterator.size > 1)
+          .mkString(s"${System.lineSeparator() * 2}")
+      }""".stripMargin
+
+  /** @param name
+    *   The name of the environment variable
+    * @param description
+    *   A short description what is it used for
+    * @param passToIde
+    *   Whether to pass this variable to the IDE/BSP client (true by default, should only be
+    *   disabled for env vars which aren't safe to save on disk)
+    * @param requiresPower
+    *   Whether this variable is related to a feature that requires power mode; also used for
+    *   internal toggles and such
+    */
+  case class EnvVar(
+    name: String,
+    description: String,
+    passToIde: Boolean = true,
+    requiresPower: Boolean = false
+  ) {
+    def valueOpt: Option[String]  = Option(System.getenv(name))
+    override def toString: String = s"$name=${valueOpt.getOrElse("")}"
+    def helpMessage(spaces: String): String = {
+      val powerString =
+        if requiresPower then s"${ScalaCliConsole.GRAY}(power)${Console.RESET} " else ""
+      s"${Console.YELLOW}$name${Console.RESET}$spaces$powerString$description"
+    }
+  }
+  object EnvVar {
```

Comment:
Do we need to keep it in `EnvUtil` ? It causes each access to have `EnvUtil.EnvVar` which is a bit redundant

Reply:
moved everything to the `EnvVar` companion object.